app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import { Checkbox, Avatar, Button, Tooltip } from "neetoui";
+
+function ContactsTable({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+  onCLickDelete,
+}) {
+  return (
+    <div className="w-full px-4">
+      <table className="nui-table nui-table--checkbox">
```

Comment:
Please add classnames `nui-table--hover nui-table--actions`. 

These classnames add the required hover effects to the table row and IconButtons.